src/main/java/decathlon/Decathlon.java

```diff
@@ -0,0 +1,42 @@
+package decathlon;
+
+import calculation.PlaceCalculator;
+import converter.XMLExporter;
+import data.Athlete;
+import fileutils.CSVReader;
+import parser.AthletesParser;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class Decathlon {
+
+    public static void main(String[] args) throws IOException {
+        if (args[0] == null && args[1] == null) {
```

Comment:
if args.length <= 2?
Because you have three ifs that do the same thing and its not IndexOutOfBoundsException-safe